src/components/PageSectionHeader/PageSectionHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import { Typography } from "@material-ui/core";
+import VerticalSpacer from "@saleor/apps/components/VerticalSpacer";
+import React from "react";
+
+interface PageSectionHeaderProps {
+  title?: React.ReactNode;
```

Comment:
Since this is a header, why is title optional?

Reply:
The component could be used in other settings pages, where title is not set

Reply:
Did @pwgryglak confirm there's such plan?

Reply:
yes there is

Reply:
Okay 👍